source/_components/binary_sensor.august.markdown

```diff
@@ -2,14 +2,14 @@
 layout: page
 title: "August Binary Sensor"
 description: "Instructions on how to integrate your August devices into Home Assistant."
-date: 2018-02-17 22:00
+date: 2018-10-03 15:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: august.png
-ha_category: Doorbell
-ha_release: "0.64"
+ha_category: Doorbell, Door
```

Comment:
Please only specify only 1 category. Secondly, `Door` isn't a valid category.

Reply:
OK, it was doorbell but really now it would be for both doorbell and door sensor. Hence why I added "door". :-)

I changed it to Binary Sensor instead as I think that is really where it probably falls under. If incorrect, can you please let me know where I can find the list of valid components?

Appreciate it!